script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -456,22 +497,22 @@ Naturally, it doesn't hurt at all.
 [sha:e8f05f4aa3016920805e50c88c6d6a74e230ac02]{
 -- Page 27, Offset 37094.
 -- そんなものより、断りもナシに振り下ろされる鈍器の方が、今は遥かに―――
-Right now, I'd much rather have this than having a blunt weapon swung at me without warning―――
+I'd take this over being mercilessly beaten over the head with a blunt object any da―――
 }
 [sha:50bbfb9ba0fb6c396b8779f4ec9bf3a5e4bb67d3]{
 -- Page 28, Offset 37095.
 -- 「―――――――、て」
-"―――――――op."
+"―――op."
 }
 [sha:4cfd47511b95dde554078ff28cd05addcc94676e]{
 -- Page 28, Offset 37096.
 -- 耐えられない。
-I can't handle this.
+I can't take it anymore.
 }
 [sha:79bfacb831ef6ecdd70843c787034c64ab3d3a53]{
 -- Page 28, Offset 37097.
 -- 気絶する事さえ、できない。
-I can't even faint.
+I can't even escape by fainting.
 }
 [sha:588dafa39990d06b28c8c47cbd103c498ce1bdea]{
 -- Page 28, Offset 37098.
```

Comment:
#Thus begins the crazy-orthography.
This sentence is a bit wrong, by the way. Maybe that was deliberate, but I don't see why it should be.
>How often does the pain come? How　 many times　per　　second――per　minute――does it　come?